### DISCUSSION TITLE
Deploying Plausible to Serverless Containers

### DISCUSSION THREAD

**Discussion by marekbrze** [plausible/analytics#1704] [Anything Else]
Hi all,

I really like [Scaleway platform](https://www.scaleway.com/en/) - I use it for the serverless functions for my static website. Right now I want to implement Plausible analytics and I am thinking about self-hosting option. 

Scaleway has a [serverless containers service ](https://www.scaleway.com/en/serverless-containers/) with cold-start containters that can scale if needed. In many other use-cases it's not a problem, but I wonder what about analytics platform like Plausible. 

Should Plausible container be always running? What about scaling when there is a high traffic on my websites and there can be more requests to the Plausible platform?

Do you have any experience with Serverless container platforms like Scaleway?

**Comment by ukutaht** [comment]
I don't recommend running on serverless. Our main app is stateful which means it's not going to work well when it's being spun up and down all the time. The session tracking is not going to work properly.

We are planning to make our services stateless to ease this pain but for now this is a constraint we live with

**Reply by marekbrze** [reply]
Thank you very much for this answer 👌

**Comment by johannbuscail** [comment]
Hey @ukutaht, any update on serverless support ?

**Reply by ruslandoga** [reply]
👋 @johannbuscail 

[Session tracking](https://github.com/plausible/analytics/blob/master/lib/plausible/session/cache_store.ex) still makes Plausible stateful.
